lib/mix/nerves/utils.ex

```diff
@@ -194,18 +194,46 @@ defmodule Mix.Nerves.Utils do
     end
   end
 
-  def get_wsl_paths(file) do
-    {win_path, 0} = System.cmd("cmd.exe", ["/c", "cd"])
-    win_path = String.trim(win_path) <> "\\#{file}"
+  def is_wslpath_available?() do
```

Comment:
There's an Elixir style to only put `is_` in functions that can be guards, so this should be named `wslpath_available?` or `has_wslpath?`

Reply:
I wasn't aware of that. I think has_wslpath? might be a better name